docs/commands/lint.md

```diff
@@ -1,77 +1,110 @@
 # `lint`
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions, with the goal of avoiding bugs and making APIs more consistent. The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
+## Introduction
 
-Use custom rules, plugins, preprocessors, and the built-in rules to define your API design standards. Every rule is a plugin that can be added at runtime.
+Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
 
+The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
 
-```mermaid
-graph LR
-    preprocessors[Preprocessors]
-    rules[Rules]
-    preprocessors --> rules
-```
+:::success Tip
+To learn more about preprocessors and rules, refer to the [custom rules](../resources/custom-rules.md) page.
+:::
 
+## Usage
 
-### `lint` usage
+```bash
+openapi lint
+openapi lint <entrypoints>...
+openapi lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
+openapi lint [--generate-ignore-file] [--help]
+openapi lint --version
+```
 
+## Options
 
-```shell
-Positionals:
-  entrypoints                                              [array] [default: []]
+Option                   | Type      | Required    | Default     | Description
+-------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoints`            | `array`   | no           | -           | Array of API definition filenames that need to be linted. See [the Entrypoints section](#entrypoints) for more options.
+`--config`               | `string`  | no           | -           | Specify path to the config file
+`--extends`              | `array`   | no           | -           | Override extends configurations (defaults or config file settings).
```

Comment:
question: How does this option work - how to use it, what does the output look like when it's used? It should have its own section with an example, like the other options do.